src/viur/shop/skeletons/cart.py

```diff
@@ -97,6 +108,14 @@ class CartNodeSkel(TreeSkel):  # STATE: Complete (as in model)
         ),
     )
 
+    total_discount_price = NumericBone(
```

Comment:
Why don't you just use total?

Reply:
Because so we can show the price with and without discount.